frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveImage/index.tsx

```diff
@@ -108,12 +110,16 @@ export default function SaveImage({
       }
       link.click();
       link.remove();
-      track(EVENTS.WMG_DOWNLOAD_COMPLETE, { file_type: fileType , tissues: selectedTissues.toString(), genes: selectedGenes.toString() });
+      track(EVENTS.WMG_DOWNLOAD_COMPLETE, {
+        file_type: fileType,
+        tissues: selectedTissues.toString(),
+        genes: selectedGenes.toString(),
```

Comment:
This output is going to be a bit diff than it has historically, might want to run this past amanda